src/components/Thought.tsx

```diff
@@ -425,6 +425,9 @@ const ThoughtContainer = ({
     [style?.textDecoration],
   )
 
+  // true when this thought is the parent of a recently dropped child, to continue a slower pulse briefly
+  const isRecentlyDropped = useSelector(state => !!state.droppedPath && equalPath(state.droppedPath, simplePath))
```

Comment:
Is `!!state.droppedPath &&` necessary? It seems like `equalPath(state.droppedPath, simplePath)` would always be false in this case anyway, since `simplePath` is always truthy.